source/linux/Foundational_Components/Kernel/Kernel_Drivers/Display/DSS7.rst

```diff
@@ -663,6 +663,59 @@ The following example was run on DSS7-UL and hence the pipe with scaling capabil
 
 Note that the ``*2`` at the end of ``-P 41@38:400x400*2`` is the scaling factor.
 
+- **Cropping**
+
+Show only a part of the frame.
+
+User can specify a frame of size ``w x h``, with an input color format ``cf``, a view ``w1 x h1`` starting at coordinates ``x1,y1`` and a plane ``w2 x h2`` to be displayed starting at coordinates ``x2,y2``. To display a cropped part of the frame(``w x h``), the below command can be used:
```

Comment:
kmstest utility can be used to demonstrate cropping in a video frame using a test pattern.
User can specify the video frame size ``w x h`` with an input color format ``cf`` using option -f,  then to specify the source rectangle i.e. to create cropped portion of video frame, it can be done using -v argument which specify ``w1 x h1`` source rectangle starting at coordinates ``x1,y1`` w.r.t the video frame. The destination rectangle i.e. the where on the screen this needs to be displayed can be specified using -p option and the desired dimensions i.e. ``w2 x h2`` to be displayed starting at coordinates ``x2,y2``.

Reply:
Updated, thanks